content/types/_http_paginated_response.textile

```diff
@@ -0,0 +1,17 @@
+An @HttpPaginatedResponse@ is a "@PaginatedResult@":#paginated-result that carries additional information about the response to an HTTP request. It is used when "making custom HTTP requests":/rest/usage#request.
+
+h4.
+  default: Properties
+  java:    Members
+  ruby:    Attributes
+  python:  Attributes
+
+Includes all properties that "@PaginatedResult@":#paginated-result has, with the additional constraint that @items@ is guaranteed to be an @Array@. Additionally, it has:
```

Comment:
I am not sure that it's really all that relevant to users that this class derives from PaginatedResult.  It's useful to provide that info perhaps, but I don't think we should expect people to jump back & forth between this documentation and the PaginatedResult documentation.  Instead duplication of the documentation would be better,